dataverse-webapp/src/main/java/edu/harvard/iq/dataverse/export/spi/Exporter.java

```diff
@@ -15,7 +15,7 @@
  */
 public interface Exporter {
 
-    String exportDataset(DatasetVersion version) throws ExportException;
+    String exportDataset(DatasetVersion version, String HideSchemaDotOrgDownloadUrls) throws ExportException;
```

Comment:
There is no need to change exporter interface, this string is only used in 1 of the exporters so changing interface for all of them is a bad practice. You can just change the constructor of the one exporter that needs this info.